src/exam/Exam.jsx

```diff
@@ -57,10 +57,12 @@ const Exam = ({
   const proctoredExamTypes = [ExamType.ONBOARDING, ExamType.PRACTICE, ExamType.PROCTORED];
 
   useEffect(() => {
-    if (examId) {
-      getProctoringSettings();
-    }
     if (proctoredExamTypes.includes(examType)) {
+      // only fetch proctoring settings for a proctored exam
+      if (examId) {
```

Comment:
moved this check inside to avoid calling the proctoring settings endpoint for timed exams (as it's no longer needed)